airflow-core/docs/howto/run-behind-proxy.rst

```diff
@@ -58,6 +58,10 @@ To do so, you need to set the following setting in your ``airflow.cfg``::
 
 - Use ``--proxy-headers`` CLI flag to tell Uvicorn to respect these headers: ``airflow api-server --proxy-headers``
 
+  Add to the API Server:
+
+        <<: *airflow-common  command: ["api-server", "--proxy-headers"]
```

Comment:
It is very unclear to users what this means. Where do I add this? How is this different from the command in the line above? More context should be added.